src/main/java/com/company/Main.java

```diff
@@ -1,32 +1,134 @@
-        System.out.println("Pls enter operator sum(+),mult(*),sub(-) or div(/)");
-        String operator = in.next();
-
-        if (operator.equalsIgnoreCase("sum") || operator.equals("+"))
-            result = value1 + value2;
-        else if (operator.equalsIgnoreCase("mult") || operator.equals("*"))
-            result = value1 * value2;
-        else if (operator.equalsIgnoreCase("sub") || operator.equals("-"))
-            result = value1 - value2;
-        else if (operator.equalsIgnoreCase("div") || operator.equals("/"))
-            result = value1 / value2;
-        else System.out.println("Wrong value, pls try again");
-
-        System.out.println("Your result  " + String.format("%.2f%n", result));
+        System.out.println("Pls enter initial value of 1st array");
+        var inValueForFirstArray = in.nextInt();
+        System.out.println("Pls enter size of arrays");
+        var sizeOfArrays = in.nextInt();
+        System.out.println("Pls enter step of 1st array");
+        var stepForFirstArray = in.nextInt();
+        System.out.println("Pls enter increment operator of 1st array (+),(*),(-) or (/)");
+        String operatorForFirstArray = in.next();
+        System.out.println("Pls enter initial value of 2nd array");
+        var inValueForSecondArray = in.nextInt();
+        System.out.println("Pls enter step of 2nd array");
+        var stepForSecondArray = in.nextInt();
+        System.out.println("Pls enter increment operator of 2nd array (+),(*),(-),(^) or (/)");
+        String operatorForSecondArray = in.next();
+        System.out.println("Pls enter operator to calculate the third array (+),(*),(-),(^) or (/)");
+        String GeneralOperator = in.next();
+        boolean x = false;
 
-    }
 
+        double[] array1 = new double[sizeOfArrays];
+        array1[0] = inValueForFirstArray;
+
+        for (int i = 1; i < sizeOfArrays; i++) {
+            switch (operatorForFirstArray) {
+                case "+":
+                    array1[i] = array1[i - 1] + stepForFirstArray;
+                    break;
+                case "-":
+                    array1[i] = array1[i - 1] - stepForFirstArray;
+                    break;
+                case "*":
+                    array1[i] = array1[i - 1] * stepForFirstArray;
+                    break;
+                case "/":
+                    array1[i] = array1[i - 1] / stepForFirstArray;
+                    break;
+                default:
+                    x = true;
+                    break;
+
+            }
+            if (x) {
+                System.out.println("Error: Wrong operator for first array");
+                break;
+
+
+            }
+        }
+        String array1String = Arrays.toString(array1);
+        System.out.println("You enter 1st array  " + array1String);
+
+
+        double[] array2 = new double[sizeOfArrays];
+        array2[0] = inValueForSecondArray;
+        for (int i = 1; i < array1.length; i++) {
```

Comment:
It would be better to refer to the size if second array since you are operating on it